lib/serialize.js

```diff
@@ -115,15 +116,14 @@ module.exports = function (app, defaults) {
       app: app,
       model: model,
       method: ctx.method.name,
+      meta: ctx.meta ? utils.clone(ctx.meta) : null,
```

Comment:
Do you need this so that you can set the meta somewhere other than afterJsonApiSerialize? (eg. in beforeJsonApiSerialize)